lib/middleware/isAdminSession.js

```diff
@@ -61,7 +61,9 @@ function isAdminUser (req, res, next) {
 						});
 
 						//set KAT session
-						req.session.kmtLoggedIn = {displayName: adminUser, userId: currentUser};
+						if (req.session) {
+							req.session.kmtLoggedIn = {displayName: adminUser, userId: currentUser};
```

Comment:
Not sure displayName is ever used. Used to show users logged in email address near licence dropdown so probably safe to remove